### ISSUE TITLE
✅make Microsoft Visual Studio 2019 :: XAML Designer not barf a hairball (Fatal System Exception) on many XAML dialogs/panels

### ISSUE THREAD

**Issue by GerHobbelt** [jimmejardine/qiqqa-open-source#40]
Double-click the XAML files and find that many fail to render with a reported Exception like Brush-blablabla-Blue-blalblBLA cannot be found, yadayada.

**Comment by GerHobbelt** [comment]
Example Exception

```
System.ArgumentException
  HResult=0x80070057
  Message='NaN' is not a valid value for property 'Value'.
  Source=WindowsBase
  StackTrace:
   at System.Windows.DependencyObject.SetValueCommon(DependencyProperty dp, Object value, PropertyMetadata metadata, Boolean coerceWithDeferredReference, Boolean coerceWithCurrentValue, OperationType operationType, Boolean isInternal)
   at System.Windows.DependencyObject.SetValue(DependencyProperty dp, Object value)
   at System.Windows.Controls.Primitives.RangeBase.set_Value(Double value)
   at Qiqqa.Brainstorm.SceneManager.SceneRenderingControlScrollWrapper.DoHorizontal() in W:\Users\Ger\Projects\sites\library.visyond.gov\80\lib\tooling\qiqqa\Qiqqa\Brainstorm\SceneManager\SceneRenderingControlScrollWrapper.xaml.cs:line 49
   at Qiqqa.Brainstorm.SceneManager.SceneRenderingControlScrollWrapper.ObjSceneRenderingControl_ScrollInfoChanged() in W:\Users\Ger\Projects\sites\library.visyond.gov\80\lib\tooling\qiqqa\Qiqqa\Brainstorm\SceneManager\SceneRenderingControlScrollWrapper.xaml.cs:line 25
   at Qiqqa.Brainstorm.SceneManager.SceneRenderingControl.FireScrollInfoChanged() in W:\Users\Ger\Projects\sites\library.visyond.gov\80\lib\tooling\qiqqa\Qiqqa\Brainstorm\SceneManager\SceneRenderingControl.xaml.cs:line 104
   at Qiqqa.Brainstorm.SceneManager.SceneRenderingControl.ViewportHasChanged() in W:\Users\Ger\Projects\sites\library.visyond.gov\80\lib\tooling\qiqqa\Qiqqa\Brainstorm\SceneManager\SceneRenderingControl.xaml.cs:line 154
   at Qiqqa.Brainstorm.SceneManager.SceneRenderingControl.SceneRenderingControl_SizeChanged(Object sender, SizeChangedEventArgs e) in W:\Users\Ger\Projects\sites\library.visyond.gov\80\lib\tooling\qiqqa\Qiqqa\Brainstorm\SceneManager\SceneRenderingControl.xaml.cs:line 697
   at System.Windows.SizeChangedEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseEvent(RoutedEventArgs e)
   at System.Windows.FrameworkElement.OnRenderSizeChanged(SizeChangedInfo sizeInfo)
   at System.Windows.ContextLayoutManager.fireSizeChangedEvents()
   at System.Windows.ContextLayoutManager.UpdateLayout()
   at System.Windows.UIElement.UpdateLayout()
   at System.Windows.Interop.HwndSource.SetLayoutSize()
   at System.Windows.Interop.HwndSource.set_RootVisualInternal(Visual value)
   at System.Windows.Interop.HwndSource.set_RootVisual(Visual value)
   at Microsoft.VisualStudio.DesignTools.DesignerContract.Isolation.Remoting.RemoteUIElement.<>c__DisplayClass20_0.<Microsoft.VisualStudio.DesignTools.DesignerContract.Isolation.Remoting.IRemoteUIElement.CreateContent>b__1(RemoteUIElement e)
```

**Comment by GerHobbelt** [comment]
Above example is fixed in commit SHA-1: af670a88f8fb56d090ed8d04bfb9b08cb0e53b33

**Comment by GerHobbelt** [comment]
Fixed as per #33 - not released as setup.exe binary yet.

**Comment by GerHobbelt** [comment]
> Closing <sup>and decluttering the issue list so it stays workable for me</sup>: fixed in https://github.com/GerHobbelt/qiqqa-open-source mainline=master branch, pending #15 / any maintainer rights/actions.
